### PR TITLE
Feat/v3nki-profile view template

### PR DESCRIPTION
**Pull request by venkateshwarv**
Profile widget can also use templateID to fetch a profile view.

Profile view are fetched by longpolling mechanism and transform the data returned.

Updated SAS tokens for all widgets.

### CODE REVIEW

File: ProfileView/src/index.ts [hunk 1]
```diff
@@ -171,13 +189,82 @@ export default class CjaasProfileWidget extends LitElement {
...
+          Authorization: "SharedAccessSignature " + this.profileReadToken,
+        },
+      })
+        .then((x) => x.data)
+        .then((x) => {
+          if (x.runtimeStatus === "Completed") {
+            this.showSpinner = false;
+            clearInterval(intervalId);
+            this.profile = x?.output?.ProfileView?.AttributeView.$values.map(
+              (x: any) => {
```
Comment (futurethang) [inline ProfileView/src/index.ts:241]: Maybe nitpicky, but is there a more descriptive term than `x` that could be used here? I always find that to be a little too abstract and adds time to reading later . . . unless the case is that it truly is such an undetermined dataset as to have no better name.

File: ProfileView/src/[sandbox]/sandbox.ts [hunk 2]
```diff
@@ -92,16 +92,17 @@ export class Sandbox extends LitElement {
...
-              stream-read-token="so=demoassure&sn=sandbox&ss=stream&sp=r&se=2022-06-17T19:18:05.538Z&sk=sandbox&sig=nJOri1M66leDMnfL93UlufHegDf3hAwoQ%2FMj37ReQBs%3D"
+              template-id="my-new-template"
+              customer="560021-Venki"
+              profile-read-token="so=demoassure&sn=sandbox&ss=profile&sp=r&se=2024-07-03T17:12:28.082618200Z&sk=sandbox&sig=NiOhZ13iRNK6h1wkdNXfHSZ6abo%2FKhvVj7LE60a%2F7Lg%3D"
+              profile-write-token="so=demoassure&sn=sandbox&ss=profile&sp=w&se=2049-02-22T16:45:11.988Z&sk=sandbox&sig=KLtI6vyZXMvvymFjqSR2EOJzeULzm43T6y3HTf63qus%3D"
+              tape-read-token="so=demoassure&sn=sandbox&ss=tape&sp=r&se=2049-02-22T16:44:19.899Z&sk=sandbox&sig=zfaMLDT15AoCvrWle0HSezBwgETtXs5JMTMQDaIkWkQ%3D"
+              stream-read-token="so=demoassure&sn=sandbox&ss=stream&sp=r&se=2049-02-22T16:45:54.572Z&sk=sandbox&sig=Ao2mLkSfBmnQH%2B87LvkIrulx61Bpb5fxFch6lwOLu78%3D"
               timelineType="journey-and-stream"
-              base-url="https://uswest-nonprod.cjaas.cisco.com"
+              base-url="https://cjaas-devus1-edge.azurewebsites.net"
```
Comment (elenagiere) [inline ProfileView/src/[sandbox]/sandbox.ts:102]: do we want to save this base-url change? or in the end have it remain uswest-nonprod?

File: ProfileView/src/index.ts (hunk 1 above)
Comment (venkateshwarv) [reply ProfileView/src/index.ts:241]: Agreed. I will amend this.

File: ProfileView/src/index.ts [hunk 3]
```diff
@@ -171,13 +189,82 @@ export default class CjaasProfileWidget extends LitElement {
...
+      headers: {
+        "Content-type": "application/json",
+        Authorization: "SharedAccessSignature " + this.profileReadToken,
+        "X-CACHE-MAXAGE-HOUR": 5,
+      },
+    };
+
+    axios(options)
+      .then((x) => x.data)
+      .then((x) => {
```
[the remark this answers is not in the document]
Comment (venkateshwarv) [reply ProfileView/src/index.ts:216]: Lambda functions with one liner are easier to read with x or y.

When there are more operations, we can use a descriptive name.

**Pull request merged**